core/src/rpc/aggregator.rs

```diff
@@ -222,26 +187,26 @@ impl ClementineAggregator for Aggregator {
             tx.send(deposit_finalize_first_param.clone()).await.unwrap();
         }
 
-        for nonce_idx in 0..num_required_nonces {
+        let sighash_stream = create_nofn_sighash_stream(
+            self.db.clone(),
+            deposit_outpoint,
+            evm_address,
+            recovery_taproot_address,
+            user_takes_after,
+            nofn_xonly_pk,
+        );
+        pin_mut!(sighash_stream); // needed for iteration
```

Comment:
From Github repo (https://github.com/rust-lang/pin-utils?tab=readme-ov-file):

> pin_utils::pin_mut! is soft-deprecated in favor of [pin! macro in the standard library](https://doc.rust-lang.org/std/pin/macro.pin.html) that stabilized in Rust 1.68.

Reply:
Fixed in https://github.com/chainwayxyz/clementine/pull/361/commits/8e24d02b1eae30d5f0382118bf11c63142f838df